tests/test_e2e_40_sdntrace.py

```diff
@@ -772,3 +772,172 @@ def test_060_run_sdntrace_order(cls):
         data = response.json()
         list_results = data["result"]
         assert list_results[0][0]["out"]["port"] == 1
+
+    def test_070_run_sdntrace_untagged_vlan(cls):
+        """Run SDNTrace to test run_sdntrace when vlan is untagged"""
+        payload_stored_flow = {
+            "flows": [
+                {
+                    "priority": 100,
+                    "match": {
+                        "in_port": 2,
+                        "dl_vlan": 0
+                    },
+                    "actions": [
+                        {
+                            "action_type": "output",
+                            "port": 1
+                        }
+                    ]
+                },
+                {
+                    "priority": 10,
+                    "match": {
+                        "in_port": 2
+                    },
+                    "actions": [
+                        {
+                            "action_type": "output",
+                            "port": 2
+                        }
+                    ]
+                }
+            ]
+        }
+        api_url = KYTOS_API + '/kytos/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
+        response = requests.post(api_url, json = payload_stored_flow)
+        assert response.status_code == 202, response.text
+        time.sleep(10)
+
+        payload = [
+                    {
+                        "trace": {
+                            "switch": {
+                                "dpid": "00:00:00:00:00:00:00:01",
+                                "in_port": 2
+                            },
+                            "eth": {
```

Comment:
We shouldn't be testing or acceting "dl_vlan": "untagged" as mentioned in the related code review. Could you remove the "eth" header here? That way it's expected to still pass. `dl_vlan: 0` in the eth header payload we can still accept though, it's supposed to be equivalent to when eth header isn't present.

Reply:
Done!